File: java/media_ponderada.java
import java.util.Locale;
import java.util.Scanner;

public class media_ponderada {

	public static void main(String[] args) {

		Locale.setDefault(Locale.US);
		Scanner sc = new Scanner(System.in);

		int N;
		double valor1, valor2, valor3, media;

		System.out.print("Quantos casos voce vai digitar? ");
		N = sc.nextInt();

		for (int i = 0; i < N; i++) {
			System.out.println("Digite tres numeros:");
			valor1 = sc.nextDouble();
			valor2 = sc.nextDouble();
			valor3 = sc.nextDouble();

			media = (valor1 * 2.0 + valor2 * 3.0 + valor3 * 5.0) / 10.0;

			System.out.printf("MEDIA = %.1f\n", media);
		}

		sc.close();
	}
}

File: python/media_ponderada.py
N: int
valor1: float
valor2: float
valor3: float
media: float

N = int(input("Quantos casos voce vai digitar? "))

for i in range(N):
    print("Digite tres numeros:")
    valor1 = float(input())
    valor2 = float(input())
    valor3 = float(input())

    media = (valor1 * 2.0 + valor2 * 3.0 + valor3 * 5.0) / 10.0

    print(f"MEDIA = {media:.1f}")
